benchmarks/portable/full_suite.py

```
@@ -331,14 +374,704 @@ def write_chaos_exports(summary: dict[str, dict], timestamp: str) -> None:
+                    )
+                )
+            else:
+                latency_rows.append(
+                    (
+                        "",
+                        [
+                            html.escape(scenario),
+                            f"{values.get('mean_p50_us', 0) / 1000:,.2f}",
+                            f"{values.get('mean_p95_us', 0) / 1000:,.2f}",
+                            f"{values.get('mean_p99_us', 0) / 1000:,.2f}",
+                            f"{values.get('min_p50_us', 0) / 1000:,.2f} – {values.get('max_p50_us', 0) / 1000:,.2f}",
```

Ruff (0.15.10)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)